<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="GOT500VI73546" ignored="0" name="GB(failed)(failed)(failed)" tests="1" failures="0" timestamp="23 Jun 2018 11:13:10 GMT" time="44.444" errors="0">
  <testcase name="@AfterClass cleanUp" time="0.0" classname="com.volvo.ncl.test.TestBase">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at com.volvo.ncl.test.TestBase.cleanUp(TestBase.java:221)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
... Removed 11 stack frames]]>
    </failure>
  </testcase> <!-- @AfterClass cleanUp -->
  <testcase name="@AfterTest endTest" time="0.0" classname="com.volvo.ncl.test.TestBase">
    <skipped/>
  </testcase> <!-- @AfterTest endTest -->
  <testcase name="DealerSelection" time="32.496" classname="com.volvo.ncl.test.TestDealerHome"/>
</testsuite> <!-- GB(failed)(failed)(failed) -->
